extensions/panache/hibernate-orm-panache-kotlin/runtime/pom.xml

```diff
@@ -21,6 +21,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-hibernate-orm</artifactId>
         </dependency>
+        <dependency>
```

Comment:
do the `mongodb-panache-kotlin` poms need updating or do they already dep on `quarkus-kotlin`?